src/test/java/gov/nih/nci/evs/api/controller/ConceptControllerTests.java

```diff
@@ -123,17 +123,6 @@ public void testGetConcept() throws Exception {
         .isEqualTo(0);
     assertThat(concept.getSynonyms().stream().filter(s -> s.getStemName() != null).count())
         .isEqualTo(0);
-    assertThat(concept.getSynonyms().stream().filter(s -> s.getTypeCode() != null).count())
-        .isEqualTo(0);
-    assertThat(concept.getDefinitions().stream().filter(s -> s.getCode() != null).count())
-        .isEqualTo(0);
-    assertThat(concept.getAssociations().stream().filter(s -> s.getCode() != null).count())
-        .isEqualTo(0);
-    assertThat(concept.getInverseAssociations().stream().filter(s -> s.getCode() != null).count())
-        .isEqualTo(0);
-    assertThat(concept.getRoles().stream().filter(s -> s.getCode() != null).count()).isEqualTo(0);
-    assertThat(concept.getInverseRoles().stream().filter(s -> s.getCode() != null).count())
-        .isEqualTo(0);
   }
```

Comment:
looks good.  I think the other thing would be to see if any of the properties or synonyms (or anything) else in the concept being tested here has a qualifier so you can check qualifier codes also.